src/util/get-github-client-config.ts

```diff
@@ -17,46 +17,43 @@ export interface GitHubClientConfig {
 	baseUrl: string;
 }
 
-export async function getGitHubApiUrl(jiraHost: string) {
-	const gitHubClientConfig = await getGitHubClientConfig(jiraHost);
-
+export async function getGitHubApiUrl(gitHubInstallationId: number, jiraHost: string) {
+	const gitHubClientConfig = await getGitHubClientConfig(gitHubInstallationId);
 	return await booleanFlag(BooleanFlags.GHE_SERVER, false, jiraHost) && gitHubClientConfig
 		? `${gitHubClientConfig.baseUrl}`
 		: GITHUB_CLOUD_API_BASEURL;
 }
 
-const getGitHubClientConfig = async (jiraHost: string): Promise<GitHubClientConfig> => {
-	// TODO: remove duplicate jiraHosts and find issue that causes duplicates
-	const installation = await Installation.getForHost(jiraHost);
-	const gitHubAppId = installation?.githubAppId;
+const getGitHubClientConfig = async (gitHubInstallationId: number): Promise<GitHubClientConfig> => {
```

Comment:
No, it will never be empty. A `gitHubInstallationId` is created when an entry is made in the Subscriptions table and is not a new field. The `gitHubInstallationId` is unique to the org where the app has been installed.